EIPS/eip-5744.md

```diff
@@ -0,0 +1,95 @@
+---
+eip: 5744
+title: Latent Fungible Token
+description: An interface for tokens that become fungible after a period of time.
+author: Cozy Finance (@cozyfinance), Tony Sheng (@tonysheng), Matt Solomon (@mds1), David Laprade (@davidlaprade), Payom Dousti (@payomdousti), Chad Fleming (@chad-js), Franz Chen (@Dendrimer)
+discussions-to: https://ethereum-magicians.org/t/eip-5744-latent-fungible-token/11111
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-09-29
+requires: 20, 2612
+---
+
+## Abstract
+
+The following standard is an extension of [EIP-20](./eip-20.md) that enables tokens to become fungible after some initial non-fungible period.
+Once minted, tokens are non-fungible until they reach maturity.
+At maturity, they become fungible and can be transferred, traded, and used in any way that a standard EIP-20 token can be used.
```

Comment:
IIUC you are saying confusion may stem from defining tokens as initially "non-fungible", and the worry is that this may be conflated with EIP-721, which is unrelated aside from the fact that they both use the adjective "non-fungible"?

If so, how you recommend rephrasing this? Perhaps adding a sentence explaining that EIP-721 is for tokens that are permanently non-fungible, and is therefore unrelated?

Reply:
Yep, that's exactly my concern. I think calling these "latent fungible tokens" implies that up until the unlock time, the tokens are non-fungible instead of non-transferable.